examples/todomvc/routes.js

```diff
@@ -0,0 +1,10 @@
+import { StateInfoConfig } from 'navigation'
+
+function configureStateInfo() {
+  StateInfoConfig.build([
+    { key: 'todomvc', initial: 'app', states: [
+      { key: 'app', route: '{filter?}', defaults: { filter: 'all' }, trackCrumbTrail: false }
+    ] }   
+  ])
+}
+export default configureStateInfo
```

Comment:
Builds the route configuration. There’s one route with an optional filter parameter that has a default value of 'all'